Graphs/testHelper.js

```diff
@@ -61,24 +61,25 @@ export const checkSvg = (component) => {
 }
 
 export const totalRows = ($) => {
-    const noOfRows = $('table tbody tr').length;
+    const noOfRows = $('.ReactVirtualized__Table').attr('aria-rowcount');
     return noOfRows;
 }
 
 export const totalColumn = ($) => {
-    const noOfColumns = $('table').find('thead').find('tr').children().length;
+    const noOfColumns = $('.ReactVirtualized__Table').attr('aria-colcount');
     return noOfColumns;
 }
 
 export const checkRowData = ($, rowNo) => {
```

Comment:
Check my comment in the other PR: https://github.com/nuagenetworks/vis-graphs/pull/462#discussion_r627713223
We should not pass hard-coded indices, instead compute the object in the test and pass the object. That way we can reuse this function without having to add multiple if statements.

Reply:
Done.